index.html

```diff
@@ -4,7 +4,63 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
+    <link rel="stylesheet" href="css/style.css" />
+
     <title>Portfolio</title>
   </head>
-  <body></body>
+  <body>
+    <header>
+      <div>
+        <h1 class="logo">My Logo</h1>
+      </div>
+      <div>
+        <span class="bars">
+          <i class="fa-solid fa-bars"></i>
+        </span>
+      </div>
+    </header>
+    <div class="container">
+      <section>
+        <div>
+          <h2 class="hero-main-text">I'm Mariane Glad to see you!</h2>
+          <h4 class="hero-minor-text">
+            I'm a software! I can help you build a product, feature or website
+            Look through some of my work and experience! If you like what you
+            see and have a project you need coded, don't hestiate to contact me.
+          </h4>
+        </div>
+        <div>
+          <div class="text-connect">
+            <h5>LET'S CONNECT</h5>
+          </div>
+          <div>
+            <div class="icons-hero">
+              <span>
+                <img src="./img/twitter.svg" alt="twitter logo" />
+              </span>
+              <span>
+                <img src="./img/linkedin.svg" alt="linkedin logo" />
+              </span>
+              <span>
+                <img src="./img/medium.svg" alt="medium logo" />
+              </span>
+              <span>
+                <img src="./img/github.svg" alt="github logo" />
+              </span>
+              <span>
+                <img src="./img/angellist.svg" alt="angellist logo" />
+              </span>
+            </div>
```

Comment:
- [ ] When listing items, you should use the ul and li tags because it is semantic. It is easy for screen readers to parse through them and those who use it will understand what the element is meant for. Inside each li, kindly use the a tag since these are social media icons and ideally, they should link to your social media pages. In the end, the structure of HTML for these subsections should be like this:

```HTML
<ul>
                    <li class="list-item"><a href="#"><img src="...", alt="..." /></a></li>
                    <li class="list-item"><a href="#"><img src="...", alt="..." /></a></li>
                    <li class="list-item"><a href="#"><img src="...", alt="..." /></a></li>
                    ...
                </ul>
```